PowerEditor/src/menuCmdID.h

```diff
@@ -597,6 +597,11 @@
     #define  IDM_WINDOW_SORT_FS_ASC	(IDR_WINDOWS_MENU + 8)
     #define  IDM_WINDOW_SORT_FS_DSC	(IDR_WINDOWS_MENU + 9)
     #define  IDM_WINDOW_MRU_FIRST 	(IDR_WINDOWS_MENU + 20)
-    #define  IDM_WINDOW_MRU_LIMIT 	(IDR_WINDOWS_MENU + 29)

```

Comment:
now it is possible to use it beyond 10 items
![image](https://user-images.githubusercontent.com/55940305/171987793-b4f0691f-3ba8-4a42-83d0-fa5b62f70b0a.png)
if there is only one number shortcut it will select it immediately, if more pressing the same key number will switch between them, to select press `ENTER`

Reply:
It's not the same behaviour.
- The current behaviour: just type the number than it works.
- This PR's behaviour: user has to hold ALT + the number.

I would say let's keep the current behaviour for 1-9, and use the PR's behaviour for the numbers beyond 9. 
Is it possible?